typescript-advance/exercise-1.js

```diff
@@ -0,0 +1,99 @@
+function createObserver() {
```

Comment:
I believe we shouldn't include js file under git
Please, remove this file under git.

Reply:
File removed. Thank you for the notice.